sale_fixed_discount/views/sale_portal_templates.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="sale_order_portal_content"
+        inherit_id="sale.sale_order_portal_content"
+        priority="100"
+    >
+        <xpath
+            expr="//section[@id='details']//t[@t-set='display_discount']"
+            position="after"
+        >
+            <t
+                t-set="display_discount_fixed"
+                t-value="True in [line.discount_fixed > 0 for line in sale_order.order_line]"
+            />
+        </xpath>
+        <xpath
+            expr="//section[@id='details']//table/thead//th[@t-if='display_discount']"
+            position="before"
+        >
+            <th
+                t-if="display_discount_fixed"
+                t-attf-class="text-end {{ 'd-none d-sm-table-cell' if report_type == 'html' else '' }}"
+            >
+                <span>Disc. Fixed Amount</span>
+            </th>
+        </xpath>
+        <xpath
+            expr="//section[@id='details']//tbody[hasclass('sale_tbody')]//t[@t-foreach='lines_to_report']/tr//t[@t-if='not line.display_type']//td[3]/div"
+            position="replace"
+        >
+            <div
+                t-if="line.discount &gt;= 0 or line.fixed_discount &gt;= 0"
+                t-field="line.price_unit"
+                t-att-style="(line.discount or line.discount_fixed) and 'text-decoration: line-through' or None"
+                t-att-class="((line.discount or line.discount_fixed) and 'text-danger' or '') + ' text-right'"
+            />
+            <div t-if="line.discount_fixed">
+                <t
+                    t-esc="line.price_unit - line.discount_fixed"
+                    t-options='{"widget": "float", "decimal_precision": "Product Price"}'
+                />
+            </div>
+        </xpath>
+        <xpath
+            expr="//section[@id='details']//table/tbody//t[@t-foreach='lines_to_report']/tr//t[@t-if='not line.display_type']//td[@t-if='display_discount']"
```

Comment:
I also get here an error and replaced it with:
`expr="//section[@id='details']//t[@t-if='not line.display_type']//td[@t-if='display_discount']"`